src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -1049,12 +1049,13 @@ public final byte[] getIV() {
      * Returns the parameters used with this cipher.
      *
      * <p>The returned parameters may be the same that were used to initialize
-     * this cipher, or may contain a combination of default and random
-     * parameter values used by the underlying cipher implementation if this
-     * cipher requires algorithm parameters but was not initialized with any.
+     * this cipher, or may contain additional default or random parameter
+     * values used by the underlying cipher implementation. If the required
+     * parameters were not supplied and the underlying cipher implementation
+     * can generate the parameter values, it will be returned. Otherwise,
```

Comment:
"it will be returned" sounds a bit awkward here. I would change this to "the generated parameter values will be returned".

Reply:
Ok.